extensions/wikia/FliteTag/FliteTagController.class.php

```diff
@@ -0,0 +1,94 @@
+<?php
+class FliteTagController extends WikiaParserTagController {
+	const MIN_SIZE = 1;
+	const PARSER_TAG_NAME = 'flite';
+	const GUID_PATTERN = '/[a-z0-9]{8,8}\-[a-z0-9]{4,4}\-[a-z0-9]{4,4}\-[a-z0-9]{4,4}\-[a-z0-9]{12,12}/';
```

Comment:
It can be simplified, e.g.
`{8,8}` => `{8}`
And I'm not sure if it is full or a part of GUID? Can we add `^` and `$` to make it safer?